File: 062.UniquePaths/clywin123/uniquePaths.py
class Solution:
    # @param {integer} m
    # @param {integer} n
    # @return {integer}
    def uniquePaths(self, m, n):
        mappath = [[0 for i in range(n)] for j in range(m)]
        for i in range(n):
            mappath[0][i] = 1
        for i in range(m):
            mappath[i][0] = 1
        for i in range(1,m):
            for j in range(1,n):
                mappath[i][j]=mappath[i-1][j]+mappath[i][j-1]
        return mappath[m-1][n-1]

File: 062.UniquePaths/wyrj/UniquePaths.js
/**
 * @param {number} m
 * @param {number} n
 * @return {number}
 */
var uniquePaths = function(m, n) {
    var table = [], i, j;
    for (i = 0; i < m; i++) {
        table[i] = [];
        for (j = 0; j < n; j++) {
            if (0 === i || 0 === j) {
                table[i][j] = 1;
            } else {
                table[i][j] = table[i - 1][j] + table[i][j - 1];
            }
        }
    }
    return table[m - 1][n - 1];
};